src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseContent.tsx

```diff
@@ -166,28 +190,15 @@ const ReleaseContent = () => {
                   View data and files
                 </a>
               </li>
-              {release.publication.methodologies.map(methodology => (
-                <li key={methodology.id}>
+              {allMethodologies.map(methodology => (
+                <li key={methodology.key} className="govuk-!-margin-bottom-9">
```

Comment:
This seems like quite a lot of bottom margin, why are we adding this?

Reply:
I've tried this out now and we don't need any padding at all so I've removed it. Looked a bit strange before. 🙂